### PR TITLE
AJAX Functions Added

### PR DESCRIPTION
**Pull request by OppenheimerAndTheMartians**

### CODE REVIEW

File: client/src/components/App.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import Landingpage from './Landingpage.js';
+import Landingpage from './LandingPage.js';
```
Comment (bentljm) [inline client/src/components/App.js]: change back to Landingpage

File: client/src/utils/AppRouters.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    success: function(data) {
+      console.log("SUCCESS: OBTAINED ALL USERS: " + data)
+    };
+    error: function(err) {console.log("ERROR: ")})
+}
+
+function getReflectionId () {
+  $.ajax({
+    type: 'GET',
+    url: '/api/users/:username/goals/:goal_id/reflections/:reflection_id,
```
Comment (bentljm) [inline client/src/utils/AppRouters.js]: missing ' after reflection_id

File: server/routers.js [hunk 3]
```diff
@@ -23,13 +23,13 @@ module.exports = function(app, express) {
 
   app.get('/api/users/:username/goals/:goal_id/reflections/:reflection_id', controller.getReflectionId); // Get individual reflection id
 
-  app.post('/api/users/:username/goals/:goal_id/reflections', controller.postReflectionId); // Post individual reflection id
+  app.post('/api/users/:username/goals/:goal_id/reflections/:reflection_id', controller.postReflectionId); // Post individual reflection id
```
Comment (bentljm) [inline server/routers.js]: remove last parameter

File: server/routers.js [hunk 4]
```diff
@@ -23,13 +23,13 @@ module.exports = function(app, express) {
...
 
   app.get('/api/users/:username/goals', controller.getAllGoals); //Get all goals
 
   app.get('/api/users/:username/goals/:goal_id', controller.getSingleGoal); //Get single goal
 
-  app.post('/api/users/:username/goals', controller.postSingleGoal); //Post single goal
+  app.post('/api/users/:username/goals/:goal_id', controller.postSingleGoal); //Post single goal
```
Comment (bentljm) [inline server/routers.js]: same thing here

File: server/server.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ var express = require('express');
 var bodyparser = require('body-parser');
 var path = require('path');
 var router = require('./routers.js');
+var router = require('./routers.js')
```
Comment (Josephine-Chen) [inline server/server.js]: Extra router

**Comment by Josephine-Chen** [comment]
Not ready to be merged to master. Needs significant overhaul.

**Pull request closed without merging**